v2/support/requirements_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+  </head>
+  <style>
+    * { font-family: sans-serif; font-size: small; }
+  </style>
+  <body>
+    <p>The UI of your app with requirements can go in here.</p>
+    <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+    <script>
+      var client = ZAFClient.init();
+      client.on('app.registered', function(data) {
```

Comment:
@fvzzy would you mind writing some code to demonstrate the [`requirement`](https://github.com/zendesk/zaf_docs/pull/139/files#diff-454ec231cb6e43d820f5dc4811d38593R44) API here? Ideally this app would create a ticket field and demonstrate retrieving that ticket field's value. Perhaps we could listed for the `changed` event and update the UI with that field's value whenever it changes?